src/pages/ProfilesExplorerView/components/SceneExploreServiceLabels/components/SceneGroupByLabels/components/SceneLabelValuesGrid/SceneLabelValuesGrid.tsx

```diff
@@ -0,0 +1,424 @@
+
+import { EventDataReceived } from '../../../../../../domain/events/EventDataReceived';
+import { FiltersVariable } from '../../../../../../domain/variables/FiltersVariable/FiltersVariable';
+import { GroupByVariable } from '../../../../../../domain/variables/GroupByVariable/GroupByVariable';
+import { findSceneObjectByClass } from '../../../../../../helpers/findSceneObjectByClass';
+import { getSceneVariableValue } from '../../../../../../helpers/getSceneVariableValue';
+import { buildTimeSeriesQueryRunner } from '../../../../../../infrastructure/timeseries/buildTimeSeriesQueryRunner';
+import { SceneEmptyState } from '../../../../../SceneByVariableRepeaterGrid/components/SceneEmptyState/SceneEmptyState';
+import { SceneErrorState } from '../../../../../SceneByVariableRepeaterGrid/components/SceneErrorState/SceneErrorState';
+import {
+  LayoutType,
+  SceneLayoutSwitcher,
+  SceneLayoutSwitcherState,
+} from '../../../../../SceneByVariableRepeaterGrid/components/SceneLayoutSwitcher';
+import {
+  SceneNoDataSwitcher,
+  SceneNoDataSwitcherState,
+} from '../../../../../SceneByVariableRepeaterGrid/components/SceneNoDataSwitcher';
+import { PanelType } from '../../../../../SceneByVariableRepeaterGrid/components/ScenePanelTypeSwitcher';
+import {
+  SceneQuickFilter,
+  SceneQuickFilterState,
+} from '../../../../../SceneByVariableRepeaterGrid/components/SceneQuickFilter';
+import { sortFavGridItems } from '../../../../../SceneByVariableRepeaterGrid/domain/sortFavGridItems';
+import {
+  addRefId,
+  addStats,
+  sortSeries,
+} from '../../../../../SceneByVariableRepeaterGrid/infrastructure/data-transformations';
+import { GridItemData } from '../../../../../SceneByVariableRepeaterGrid/types/GridItemData';
+import { SceneLabelValuePanel } from './components/SceneLabelValuePanel';
+
+export interface SceneLabelValuesGridState extends EmbeddedSceneState {
+  $data: SceneDataProvider;
+  isLoading: boolean;
+  items: GridItemData[];
+  label: string;
+  startColorIndex: number;
+  headerActions: (item: GridItemData, items: GridItemData[]) => VizPanelState['headerActions'];
+  sortItemsFn: (a: GridItemData, b: GridItemData) => number;
+  hideNoData: boolean;
+}
+
+const GRID_TEMPLATE_COLUMNS = 'repeat(auto-fit, minmax(800px, 1fr))';
+const GRID_TEMPLATE_ROWS = '1fr';
+export const GRID_AUTO_ROWS = '160px';
+
+export class SceneLabelValuesGrid extends SceneObjectBase<SceneLabelValuesGridState> {
+  static buildGridItemKey(item: GridItemData) {
+    return `grid-item-${item.index}-${item.value}`;
+  }
+
+  constructor({
+    key,
+    label,
+    startColorIndex,
+    headerActions,
+  }: {
+    key: string;
+    label: SceneLabelValuesGridState['label'];
+    startColorIndex: SceneLabelValuesGridState['startColorIndex'];
+    headerActions: SceneLabelValuesGridState['headerActions'];
+  }) {
+    super({
+      key,
+      label,
+      startColorIndex,
+      items: [],
+      isLoading: true,
+      $data: new SceneDataTransformer({
```

Comment:
This is one of the main difference with `SceneByVariableRepeaterGrid`. Using a query with the `groupBy` parameter here improves the consistency between the grid that is displayed when "All" is selected vs when a specific label has been selected. It ensures that:
- A timeseries with an empty label value is also rendered
- The panels will be sorted by decreasing total of resource consumption (thanks to the `sortSeries` data transformation)

For example, with the `vehicle` label:

<img width="1703" alt="image" src="https://github.com/user-attachments/assets/edf21fbf-3ab8-48c9-b637-dd3a5410114d">
(notice the "cpu" timeseries which corresponds to an empty label value)

And after selection, the "cpu" timeseries also appears in the grid:

<img width="1698" alt="image" src="https://github.com/user-attachments/assets/c995c107-8abc-4ef2-9e00-6945256026f2">